pyproject.toml

```diff
@@ -25,7 +25,8 @@ dependencies = [
     "sortedcontainers>=2.4.0",
     "toml>=0.10.2",
     "z3-solver==4.12.6.0",
-    "eth_hash[pysha3]>=0.7.0"
+    "eth_hash[pysha3]>=0.7.0",
+    "rich>=13.8.1"
```

Comment:
@karmacoma-eth i copied the minimum version requirement from jsi. could you confirm it's still applicable here?